flash/core/data/utils.py

```diff
@@ -18,9 +18,9 @@
 from typing import Any, Callable, Dict, Iterable, Mapping, Optional, Set
 
 import requests
-import torch
 import urllib3
 from pytorch_lightning.utilities.apply_func import apply_to_collection
+from torch.nn import Module, ModuleDict, ModuleList
```

Comment:
Thanks for the refactoring, @Borda - but I'm not very sure if this is a great idea. And this might be just me, I usually prefer having a namespace around such modules where it's not immediately clear where they come from. So something like `nn.ModuleDict` is more clear than `ModuleDict` and doing `from torch.nn import ModuleDict`.

Reply:
the long imports tend to cut lines also I believe import is slightly faster...